### PR TITLE
Intro page

### PR DESCRIPTION
**Pull request by aminata-cisse-fluxtailor**

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -25,4 +25,4 @@
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
 
 html_theme = 'sphinx_rtd_theme'
-html_static_path = ['OxygenDocs']
+html_static_path = ['OxygenDocs']
```
Comment (jonah-bossewitch-fluxtailor) [inline doc/conf.py:28]: Hmmm...  Why does this newline keep getting deleted?  Does your IDE keep deleting it?  I'm not certain if we need it or not, but some files may be sensitive.   I saw this https://community.devsense.com/d/1748-formatter-how-to-disable-removal-of-new-line-at-eof/2 and there may be a setting in Visual Studio worth double checking.

Not a big deal if this stays removed, but I wanted to figure out if this was happening automatically.

File: doc/index.md [hunk 2]
```diff
@@ -1,33 +1,38 @@
...
+---
+title: Rate Plan Schema documentation
+---
 
-## Purpose of the Rate Plan Schema Documentation
+# Welcome to the Rate Plan Schema documentation!
 
-The rate plan schema documentation serves as a comprehensive guide for understanding, implementing, and utilizing the standardized rate plan schema in the energy sector.
+```{toctree}
+:maxdepth: 2
```
Comment (jonah-bossewitch-fluxtailor) [inline doc/index.md:8]: Did we want to try a maxdepth of 3?  I think for our outline that might look better.

**Review by jonah-bossewitch-fluxtailor — CHANGES_REQUESTED** [review]
Hi @aminata-cisse-fluxtailor,

Fantastic!  Glad you got this all running smoothly.  🚀 

This PR looks really good to me except for a few small tweaks which I indicated inline.

And, can you please remove the `some-feature.md` and `another-feature.md` pages - those are becoming confusing as we build out the actual contnet.

cheers,
/Jonah

File: doc/conf.py (hunk 1 above)
Comment (aminata-cisse-fluxtailor) [reply doc/conf.py:28]: I just enabled the setting that might help with the default trimming, hopefully that works!

**Pull request merged**